docs/concepts/isis-fundamentals/gdal_support.md

```diff
@@ -0,0 +1,186 @@
+
+  Group = Field
+    Name = J2000Z
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = J2000XV
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = J2000YV
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = J2000ZV
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = ET
+    Type = Double
+    Size = 1
+  End_Group
+End_Object
+```
+
+Table Metadata and data on TIFF
+```json
+{"Table_SunPosition": 
+    '{
+        "_container_name":"Table",
+        "_type":"object",
+        "Field_J2000X":{
+            "_container_name":"Field",
+            "_type":"group",
+            "Name":"J2000X",
+            "Type":"Double",
+            "Size":"1"
+        },
+        "Field_J2000Y":{
+            "_container_name":"Field",
+            "_type":"group",
+            "Name":"J2000Y",
+            "Type":"Double",
+            "Size":"1"
+        },
+        "Field_J2000Z":{
+            "_container_name":"Field",
+            "_type":"group",
+            "Name":"J2000Z",
+            "Type":"Double",
+            "Size":"1"
+        },
+        "Field_J2000XV":{
+            "_container_name":"Field",
+            "_type":"group",
+            "Name":"J2000XV",
+            "Type":"Double",
+            "Size":"1"
+        },
+        "Field_J2000YV":{
+            "_container_name":"Field",
+            "_type":"group",
+            "Name":"J2000YV",
+            "Type":"Double",
+            "Size":"1"
+        },
+        "Field_J2000ZV":{
+            "_container_name":"Field",
+            "_type":"group",
+            "Name":"J2000ZV",
+            "Type":"Double",
+            "Size":"1"
+        },
+        "Field_ET":{
+            "_container_name":"Field",
+            "_type":"group",
+            "Name":"ET",
+            "Type":"Double",
+            "Size":"1"
+        },
+        "Name":"SunPosition",
+        "StartByte":"1",
+        "Bytes":"112",
+        "Records":"2",
+        "ByteOrder":"Lsb",
+        "CacheType":"Linear",
+        "SpkTableStartTime":"297088762.24158",
+        "SpkTableEndTime":"297088808.37074",
+        "SpkTableOriginalSize":"2.0",
+        "Description":"Created by spiceinit",
+        "Kernels":[
+            "$base/kernels/spk/de430.bsp",
+            "$base/kernels/spk/mar097.bsp"
+        ],
+        "Data":"4cffffffd401ffffffe62effffffd3ffffffa8ffffffc11d03ffffffffffffff8525495dffffffc157fffffffb03ffffff89ffffff800c404114ffffffd1ffffffa7ffffffb6ffffffecffffffe7ffffffcaffffffbfffffffc6fffffffa48ffffffd7ffffffe1ffffffe637ffffffc0246fffffff93ffffffae39ffffffea25ffffffc074ffffffd83dfffffffa36ffffffb5ffffffb141fffffff6ffffffc264fffffff92effffffd3ffffffa8ffffffc1503cffffffb32a394a5dffffffc1fffffff5fffffff746ffffffceffffff830b4041ffffffc6066d1b4fffffffe3ffffffcaffffffbf6c4f1dffffff80ffffffe1ffffffe637ffffffc03fffffffb246ffffffde39ffffffea25ffffffc0ffffff8fffffffe85e2837ffffffb5ffffffb141"
+    }'
+}
+```
```

Comment:
At any time, once cat still use `catlab` to extract the information from the ISIS-created GeoTIFF.

Is that true? Are there limitations like using `editlab`?

Reply:
I did my best to make sure any label reading or modifying app worked with the new tiffs. Good things to test once we get a dev build with the changes